settings.gradle.kts

```diff
@@ -0,0 +1,8 @@
+pluginManagement {
+    val kotlinVersion: String by settings
+    plugins {
+        kotlin("jvm") version kotlinVersion
+        kotlin("plugin.allopen") version kotlinVersion
+    }
+}
+rootProject.name = "gatling gradle plugin kotlin demo"
```

Comment:
I'm not fond of this settings:

1. we should not intercept the pluginManagement from the user's settings
2. what happens if this project is a submodule of a bigger one? does that change the real rootProject name?

Reply:
Well I do not expect the users of this sample code to include this blindly into their code base.
In case it is added to an existing code base then most likely they do already have their plugin management in place and therefore won't take this over.
But in general it is considered good practice to have this configured within the `settings.gradle.kts` and in general you'd only have one `settings.gradle.kts`. Basically you'd define the submodules inside the `settings.gradle.kts` file.

Reply:
It's a common use case now: You clone a repo to start your own module in your mono-repo.
So, I try to let the users do that.

I don't really care about pluginManagement (as I think it will be correctly handled, ie, only submodules of this one, but not siblings)
But I really don't know about this "rootProject" one.

Reply:
Agreed, I've just removed the `rootProject` part ;-)